torch/quantization/fake_quantize.py

```diff
@@ -343,6 +343,19 @@ def forward(self, X: torch.Tensor) -> torch.Tensor:
                                                       qscheme=torch.per_tensor_affine,
                                                       reduce_range=True)
 
+default_fused_act_fake_quant = FusedMovingAvgObsFakeQuantize.with_args(observer=MovingAverageMinMaxObserver,
```

Comment:
I'd vote for using `reduce_range` to be consistent with other qconfigs, and change it only when we change the other qconfigs, to keep everything consistent.  There is not clear timeline to deprecate it, and it would be confusing to users if some qconfigs used it and some did not.

Reply:
added reduce_range in #62863